resources/windows/wix/shortcuts.wxs.erb

```diff
@@ -57,7 +57,7 @@
           Target="[%ComSpec]"
           Arguments='/E:ON /K "[INSTALLDIR]bin\puppet_shell.bat"'
           Icon="PuppetShellShortcutIcon"
-          WorkingDirectory="bin">
+          WorkingDirectory="APPDATADIR">
```

Comment:
I'm 👍 though I wanted to make sure y'all are aware that it's important that untrusted users not be able to write files into the current working directory, as it can lead to dll planting attacks. In this case, puppet's appdata directory is locked down, so it's ok:

```
C:\ProgramData\PuppetLabs>icacls .
. Everyone:(OI)(CI)(RX)
  NT AUTHORITY\SYSTEM:(OI)(CI)(F)
  BUILTIN\Administrators:(OI)(CI)(F)
```